Apps/WebClient/src/ClientApp/src/components/multiSelect.vue

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+import Vue from "vue";
+import { Component, Emit, Model, Prop, Watch } from "vue-property-decorator";
+
+@Component
+export default class MultiSelectComponent extends Vue {
+    @Prop({ default: "Choose a tag..." }) placeholder: string;
+    @Prop() options: string[];
+    @Model("change", { type: Array }) public model!: string[];
+
+    private values = [];
```

Comment:
I'm getting some build errors relating to the types and initial values of the variables and props.

```
7:43 Property 'placeholder' has no initializer and is not definitely assigned in the constructor.
8:13 Property 'options' has no initializer and is not definitely assigned in the constructor.
14:65 Argument of type 'string' is not assignable to parameter of type 'never'.
24:9 Type 'string[]' is not assignable to type 'never[]'.
33:9 Type 'string[]' is not assignable to type 'never[]'.
```